test/src/kbasesearchengine/test/parse/ObjectParserTest.java

```diff
@@ -283,7 +285,7 @@ public void parseSubObjectsMediaCompoundTest() throws Exception {
         }
 
         final List<String> expectedKeys = Arrays.asList("inchikey", "concentration",
-                "minFlux", "maxFlux", "id", "name");
```

Comment:
Not a big deal, but for an exact match, expectedKeys = Arrays.asList("maxFlux", "minFlux", "name", "concentration", "id", "compound_ref")

Reply:
Is just checking the key like this ok? 
assertThat(expectedKeys.containsAll(result.keySet()), is(true));

Reply:
Sorry @JamesJeffryes I missed your question in a sea of pr emails. I think what you have is OK to get stuff out now, but if you are asking if the check is sufficient, I think it can be made more complete. To do this, your test media01.json should include all fields (required and optional), then you can check to ensure result.keySet().size() is the right size and check if the values of each of the keys is what you expect.